runtime/bref/0.2/serverless.yaml

```diff
@@ -0,0 +1,52 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: my-symfony-app
+configValidationMode: error
+useDotenv: true
```

Comment:
Unless I'm missing something you may not want to use `useDotenv: true`.

This will inject all `.env` env variables (application variables) into the `serverless` process (build/deploy process). And it will also automatically exclude `.env` files from being uploaded.

This feature was supposed to be enabled by default in Serverless Framework v3, but we've decided to keep it opt-in to avoid messing up applications. At the moment the error message says that this line should be added, but this will be changed soon.

If you want to control `.env` files separately from `serverless deploy` then you don't want to mix the 2.